tests/testPoseFactor.cpp

```diff
@@ -38,10 +38,9 @@ using gtsam::Z_6x1;
 using gtsam::noiseModel::Gaussian;
 
 namespace example {
-// nosie model
+// noise model
 Gaussian::shared_ptr cost_model = Gaussian::Covariance(gtsam::I_6x6);
-gtsam::Key wTp_key = internal::PoseKey(1), wTc_key = internal::PoseKey(2),
-           q_key = internal::JointAngleKey(1);
+size_t k = 777;
```

Comment:
Ahhh we'll have to re-add these tests again. @gchenfc man, we need to compartmentalize PRs better :/

Reply:
Actually these test changes are in #290.